src/components/ProjectSettingsTabs/GithubCommitQueueTab/getFormSchema.ts

```diff
@@ -178,6 +183,65 @@ export const getFormSchema = (
           },
         },
       },
+      commitQueue: {
+        type: "object" as "object",
+        title: "Commit Queue",
+        properties: {
+          enabled: {
+            type: ["boolean", "null"],
+            oneOf: radioBoxOptions(
+              ["Enabled", "Disabled"],
+              repoData?.commitQueue?.enabled
+            ),
+          },
```

Comment:
Sorry this additional checkbox was added more recently than the figma design:
![image](https://user-images.githubusercontent.com/26798134/152256124-d18afb27-ca80-4dac-b03e-0b5532e2ee52.png)
You can totally create a new PR to handle it, but it should work like a simple checkbox if you do want to add it here (whether this is enabled/disabled doesn't impact anything else in the section).

Reply:
Cool, it looks like this field isn't returned by GraphQL yet so I've made some new tickets to complete this: EVG-16257, EVG-16258